voskcli/transcribe.py

```diff
@@ -142,13 +142,56 @@ def transcribe(inputFile, outputFile, model):
             rec_results.append(rec.Result())
 
     rec_results.append(rec.FinalResult())
-    vtt = write_webvtt_captions(rec_results)
+    print('Finished transcribing...')
+    if punc != "":
+        print(f'Start punctuating with model {punc}')
+        # Punctuation
+        # Load text from json
+        text = ''
+        for rec_result in rec_results:
+            result = json.loads(rec_result).get('result')
+            if not result:
+                continue
+            text += ' '.join([entry['word'] for entry in result])
+
+        # Predicts Punctuation of text
+        # Manipulate main to be able to load model
+        old_main = sys.modules['__main__']
+        sys.modules['__main__'] = voskcli
+        predictor = CasePuncPredictor(punc + '/checkpoint')
```

Comment:
Trying to use this, this threw me off. Why modify the path users specified using the command line parameter `-p`? I had to look into the code to find out that the file names are hard-coded. I think just using what the user provided is less confusing to the user?